src/test/index.js

```diff
@@ -1,15 +1,68 @@
+/* globals before: false, beforeEach: false, after: false, afterEach: false, describe: false, it: false, expect: false */
 'use strict';
 
 import Storage from '../'
 import {storage as storageProxy} from '../'
-import {Waterline, HasModels, BaseModel, GeoModel} from '../'
+import {Waterline, BaseModel, GeoModel} from '../'
 
 import sinon from 'sinon'
 import {application as app} from 'nxus-core'
 
 import One from './models/One'
 import Two from './models/Two'
 
+// example from RFC 7946, section 1.5 (deconstructed and reassembled)
+const geoJSONPoint = { 'type': 'Point', 'coordinates': [102.0, 0.5] },
```

Comment:
That was my bias back in the C/C++ and Java days. Now, I'm not so sure the distinction is worth highlighting; so have tended towards declaring as `const` where appropriate, but using camel-case everywhere.